### DISCUSSION TITLE
Flows 6 point cron resource.

### DISCUSSION THREAD

**Discussion by stoberbryan** [directus/directus#14332] [Archive]
Do we have a good resource for 6 point cron formats & examples.
These 5 point formats are not working as expected and the 6 point is giving unexpected results?
https://crontab.guru/#0_22_*_*_5
https://crontab-generator.org

**Comment by benhaynes** [comment]
Beyond a resource, this seems like a _great_ custom component/interface to help guide CRON configuration.

**Comment by br41nslug** [comment]
Most systems don't support the 6th point in crons so the best resource i could find is this overview from the `node-cron` docs. Essentially you can use crontab.guru and add a point in front if that second precision is actually needed but most application schedules really dont have a need for second precision

```
 # ┌────────────── second (optional)
 # │ ┌──────────── minute
 # │ │ ┌────────── hour
 # │ │ │ ┌──────── day of month
 # │ │ │ │ ┌────── month
 # │ │ │ │ │ ┌──── day of week
 # │ │ │ │ │ │
 # │ │ │ │ │ │
 # * * * * * *
 ```
https://www.npmjs.com/package/node-cron

**Reply by stoberbryan** [reply]
#* * 9 * * 5
spaces between or no spaces?
